config/routes.rb

```diff
@@ -4,6 +4,7 @@
   resource :session
 
   get '/login' => 'sessions#new'
+  delete '/logout' => 'session#destroy'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.